src/Features/CSharp/Portable/Debugging/CSharpProximityExpressionsService.cs

```diff
@@ -100,6 +100,8 @@ public CSharpProximityExpressionsService()
             }
         }
 
+        public static IList<string> GetProximityExpressions(SyntaxTree syntaxTree, int position) => Do(syntaxTree, position, cancellationToken: default);
```

Comment:
❔ Why is this not using one of the existing methods?

Reply:
So I reutilized the existing "Do" because that was the written way to do "synchronous" expression resolution. Because of the synchronous requirement I felt using it directly was the best approach. Alternatively I could have written:
```
new Worker(syntaxTree, position).Do(cancellationToken)
```